core-rust-bridge/src/main/java/com/radixdlt/statecomputer/commit/CommitRequest.java

```diff
@@ -77,6 +78,7 @@
 public record CommitRequest(
     List<RawLedgerTransaction> transactions,
     UInt64 stateVersion,
+    HashCode stateHash,
```

Comment:
I might be tempted to call it `expectedResultantStateHash` or something, but that's also me loving long descriptive names.

Happy to keep with `stateHash` too :D